JohnsonCountyJail.yaml

```diff
@@ -882,428 +886,374 @@ entityDefinitions:
     src: "JohnsonCounty_IowaCity_JCInmates"
     dst: "JohnsonCounty_IowaCity_JCSentences"
-    entitySetName: "JohnsonCounty_IowaCity_JCSentencedWith"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
+        type: "ol.id"
+        column: "Jail_ID"
+      general.datetime:
+        type: "general.datetime"
+        column: "Date_In"
+        transforms:
+        - !<transforms.DateAsDateTimeTransform>
+          pattern: ["M/d/yyyy", "d-MMM-yy", "MM/dd/yyy", "M/dd/yyyy", "MM/d/yyyy", "dd-MMM-yy", "yyyy-MM-dd HH:mm:ss", "yyyy-MM-dd HH:mm:ss.S"]
+          timezone: "America/Chicago"
     name: "JohnsonCounty_IowaCity_JCSentencedWith1"
-  JohnsonCounty_IowaCity_JCSentencedWith2:
-    fqn: "ol.sentencedwith"
-    src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "Sentences2"
-    entitySetName: "JohnsonCounty_IowaCity_JCSentencedWith"
-    propertyDefinitions:
-      ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSentencedWith2"
-  JohnsonCounty_IowaCity_JCSubjectOf:
-    fqn: "ol.subjectof"
+
+  hasrelationshipwith:
+    fqn: "ol.hasrelationshipwith"
+    entitySetName: "JohnsonCounty_IowaCity_JCHasRelationshipWith"
     src: "JohnsonCounty_IowaCity_JCInmates"
-    dst: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCSubjectOf"
+    dst: "JohnsonCounty_IowaCity_BondRelative"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCSubjectOf"
-  JohnsonCounty_IowaCity_JCOfLength:
-    fqn: "ol.oflength"
-    dst: "JohnsonCounty_IowaCity_JCJailStayLength"
+        type: "ol.id"
+        column: "Person_Post"
+      ol.role:
+        type: "ol.role"
+        transforms:
+        - !<transforms.ValueTransform>
+          value: "Kin"
+    name: "hasrelationshipwith"
+
+  JohnsonCounty_IowaCity_JCBondSet:
+    fqn: "ol.bondset"
+    entitySetName: "JohnsonCounty_IowaCity_JCBondSet"
     src: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCOfLength"
+    dst: "JohnsonCounty_IowaCity_JCBond"
     propertyDefinitions:
       ol.id:
-        type: ol.id
-        column: Jail_ID
-    name: "JohnsonCounty_IowaCity_JCOfLength"
+        type: "ol.id"
+        column: "Jail_ID"
+    name: "JohnsonCounty_IowaCity_JCBondSet"
+
   Resultsin1:
-    fqn: 'ol.resultsin'
+    fqn: "ol.resultsin"
+    entitySetName: "JohnsonCounty_IowaCity_JCResultsIn"
     src: "JohnsonCounty_IowaCity_JCSentences"
     dst: "JohnsonCounty_IowaCity_JCJailStay"
-    entitySetName: "JohnsonCounty_IowaCity_JCResultsIn"
     propertyDefinitions:
       ol.id:
         type: "ol.id"
         transforms:
         - !<transforms.ConcatTransform>
-          columns: ['SentenceDays', "SentenceHrs", "Concurrent", "Probation", "ConsecWith","Alt_Start_Date"]
+          columns: ["SentenceDays", "SentenceHrs", "Concurrent", "Probation", "ConsecWith", "Alt_Start_Date"]
```

Comment:
should we replace Alt_Start_Date here with Start_Date since that is what is being read into the association?